### DISCUSSION TITLE
Announcement Regarding Hacktoberfest 2024 Participation - This repo is excluded from Hacktoberfest 2024!!

### DISCUSSION THREAD

**Discussion by DevAffan** [DevAffan/AffiCodes#234] [Announcements]
Hi Devs,

I apologize for the inconvenience, but I wanted to inform you that my repository has been excluded from the Hacktoberfest 2024 event. I am currently looking into the issue.

Therefore, please refrain from submitting any pull requests for Hacktoberfest participation. I've already accepted most requests, but I am now receiving queries regarding the exclusion, which happened today.

Thank you for your understanding!

**Comment by rahul-chakradhari** [comment]
but please accept my request i had pull request you sir

**Comment by DevAffan** [comment]
Repository Exclusion from Hacktoberfest – Apologies and Clarification

Hello everyone,

I wanted to take a moment to address the recent exclusion of my repository from Hacktoberfest. It has come to my attention that my repository was reported or excluded by Hacktoberfest due to a high volume of pull requests related to Data Structures and Algorithms (DSA), some of which may have included copied content.

First and foremost, I want to sincerely apologize for any confusion or inconvenience this has caused. Please know that this was not my intention, and I did not anticipate such an outcome. The purpose of this repository was to provide a meaningful contribution platform for everyone, and I fully support original, high-quality contributions.

I am working to resolve this issue and will take steps to prevent similar occurrences in the future.
Thank you for your understanding and continued support.

Best regards,
Affan Ali
